### DISCUSSION TITLE
TSA parameter file not working?

### DISCUSSION THREAD

**Discussion by kesslerf** [davidfrantz/force#219] [General]
Hi, 
I created a new TSA parameterfile using the latest docker-container and got an error stating:
HARMONIC_MODES, HARMONIC_FIT_RANGE, OUTPUT_NRT  are not set. I checked the discussions and they were mentioned in #203.
I added the values given in the prm-file that was shared there and it works, even though I'd like to know what those values actually mean/what values can be set there. 

I didn't find possible values/explanations anywhere. 

Thanks, 
Felix

**Comment by maxfreu** [comment]
The hlps section of the 3.7.8 release notes shed some light on this: https://force-eo.readthedocs.io/en/latest/history/v3/v378.html?highlight=valid#force-v-3-7-8

**Comment by kesslerf** [comment]
Thanks for the link. If I got it right it will not change my output if I set OUTPUT_NRT=FALSE?

**Reply by davidfrantz** [reply]
that's correct. I will try to fix force-parameter asap

Best,
David

**Comment by davidfrantz** [comment]
Marcel Schwieder @geo-masc was so kind to add the missing parameters + description to ``force-parameters`` (https://github.com/davidfrantz/force/commit/dd57e0c1d01ba61f80df769ce3d85b2fbe29ff21). This is included in the dev branch now.
